Source/WebCore/accessibility/mac/WebAccessibilityObjectWrapperMac.mm

```diff
@@ -3260,11 +3260,11 @@ - (id)accessibilityAttributeValue:(NSString*)attribute forParameter:(id)paramete
             if (!object)
                 return nil;
 
-            NSMutableDictionary *result = [[NSMutableDictionary alloc] initWithObjectsAndKeys:
+            RetainPtr result = adoptNS([[NSMutableDictionary alloc] initWithObjectsAndKeys:
                 object->wrapper(), @"AXSearchResultElement",
                 textMarkerRange->platformData().bridgingAutorelease(), @"AXSearchResultRange",
-                nil];
-            return [[NSArray alloc] initWithObjects:result, nil];
+                nil]);
+            return [[[NSArray alloc] initWithObjects:result.leakRef(), nil] autorelease];
```

Comment:
You don't want to use `.leakRef()` here as you'll just re-introduce the `NSMutableDictionary` leak that you fixed above. :)

Replace `.leakRef()` with `.get()`, and the compiler will take care of releasing the `NSMutableDictionary` when `result` goes out of scope.

```suggestion
            return [[[NSArray alloc] initWithObjects:result.get(), nil] autorelease];
```